content/en/api/dashboard_lists/dashboard_lists.md

```diff
@@ -7,5 +7,13 @@ external_redirect: /api/#dashboard-lists
 
 ## Dashboard Lists
 
+<div class="alert alert-info">
+
+We updated our Dashboard Lists API. Documentation for the legacy API is available here:
+    <ul>
+        <li><a href="https://docs.datadoghq.com/graphing/faq/dashboard-lists-api-doc">Dashboard Lists API doc (v1)</a></li>
+    </ul>
+</div>
+
```

Comment:
This is great and exactly what we discussed. However, when viewing the result pages, I think it is confusing because we have 9 endpoints under `Dashboard Lists`, but we only updated 4 of them so people might think all 9 were changed. I think it is better not to follow the screenboard/timebaord precisely. Instead I suggest we insert 4 notifications above each of the updated endpoints only and in each of them we have direct link directly to the section of the `v1` section. Also, we can update the wording to reflect that. For example:
This endpoint has been updated to reflect the [new Dashboard API](link to the API)  changes. Documentation for old endpoint is available here:
-Get Items of a Dashboard List (v1)( link to /graphing/guide/dashboard-lists-api-doc/?tab=python#get-items-of-a-dashboard-list)